src/pages/observability/app-builder.md

```diff
@@ -180,15 +180,3 @@ const instrumentedMain = instrumentEntrypoint(main, {
 });
```

Comment:
This example can be outdated after the latest changes in `aio-lib-telemetry`.
@iivvaannxx can you please check and provide an up-to-date example?